df_py/util/constants.py

```diff
@@ -39,3 +39,4 @@
 # predictoor
 MIN_PREDICTIONS = 1818
 DEPLOYER_ADDRS: List[str] = []
+PREDICTOOR_RELEASE_WEEK = 42  # change me when released
```

Comment:
can we say `# change me when Predictoor is released`? The wording makes me believe weird stuff I've described on Slack 😄